src/AppBundle/Controller/BehatController.php

```diff
@@ -23,6 +26,26 @@ private function securityChecks()
     }
 
     /**
+     * @Route("/client/{caseNumber}")
+     * @Method({"PUT"})
+     */
+    public function clientEditAction(Request $request, $caseNumber)
```

Comment:
Does this need additional security checks?

Reply:
it's done by `securityChecks` below. Basically completely disabled on non-prod envs